crates/sui-framework-snapshot/manifest.json

```diff
@@ -293,5 +293,15 @@
       "0x0000000000000000000000000000000000000000000000000000000000000003",
       "0x000000000000000000000000000000000000000000000000000000000000dee9"
     ]
+  },
+  "37": {
+    "git_revision": "1ca8e4c3ee-dirty",
```

Comment:
is this change supposed to be here?

Reply:
we need this for the tests

Reply:
will "dirty" be eventually removed?

Reply:
yes, will regenerate this again when we merge the feature branch to main